docs/plugins.md

```diff
@@ -261,7 +260,7 @@ This means if two transforms both visit the "Program" node, the transforms will
 - Plugin ordering is first to last.
 - Preset ordering is reversed (last to first).
 
-```js
+```json
 "plugins": [
   "transform-decorators-legacy", // will run first
```

Comment:
I think we shouldn't use comment at all in code snippets. Most of the users will remove it anyway.